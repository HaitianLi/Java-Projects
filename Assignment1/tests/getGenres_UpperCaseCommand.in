add collection books.csv
add collection books2.csv
LIST GENREs
exit