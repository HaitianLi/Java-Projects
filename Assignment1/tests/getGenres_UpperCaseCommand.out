user: 17 books successfully added.

user: 19 books successfully added.

user: Comedy
Comic
Epic
Fantasy
Genre testcase
Historical Fiction
Horror
Poetry
Test Case
Test Case 2
War

user: Ending Library process.
